### ISSUE TITLE
유튜브에서 자막 켜는 방법에 대한 매뉴얼 제공

### ISSUE THREAD

**Issue by muae** [jungtosociety/dharma-qna#9]
저는 한가지 궁금한 점이 있었는데...자막화 한 유투브를 실제로 클릭해서 보다보니, 자막을 어떻게 보시는지 모르시는 분들이 계실 것이란 생각이 들었어요

혹시 컴퓨터에 서투르신 분들께, 저희가 영어와 한글로 자막화 한 것을 보실 수 있는, 인스트럭션 영상이나 가이드가 있었으면 합니다만,
저도 처음에 oversea 동영상을 볼때, 원래 자막이 없는 줄 알고 보다가...나중에서야 여러번 클릭한 후에 찾은 것이라서...
저희 업무는 아닐지라도, 혹시 회의나 안건이 있을 떄 같이 올려주시면 감사하겠습니다 ☺

-송명석님 제안

**Comment by muae** [comment]
https://www.youtube.com/watch?v=YHuR4TIUL7U 여기서도 실제로 자막의 존재를 모르는 사람이 있는 것 같음. 지금 떠오르는 해결 방안

1) 자막 가이드 라인 동영상 추가 (어딘가에 이미 있을테니 링크라도)

2) 동영상 description에 자막이 있으니 켜라고 설명을 적어야 함.

**Comment by MisunK** [comment]
동영상 켜자마자 자막 보려면 cc 누르라는 팝업이 뜨는데요 저는? 이게 고치셔서 이렇게 된건가요?
[image: Inline images 1]

2016-10-07 7:55 GMT+11:00 muae notifications@github.com:

**Comment by muae** [comment]
네 제가 https://www.youtube.com/watch?v=YHuR4TIUL7U 에 시험삼에 annotation으로 넣어봤습니다.

**Comment by muae** [comment]
@MisunK 네 제가 https://www.youtube.com/watch?v=YHuR4TIUL7U 영상에만 시험삼아 넣어봤습니다.

**Comment by MisunK** [comment]
아하 좋네요 annotation 뜨니! 다른 영상에도 들어가도 좋을 것 같아요.
